R.swift/ResourceTypes/LocalizableStrings.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  LocalizableStrings.swift
+//  R.swift
+//
+//  Created by Nolan Warner on 2016/02/23.
+//  Copyright © 2016 Nolan Warner. All rights reserved.
+//
+
+import Foundation
+
+enum Locale {
+  case None
+  case Base
+  case Language(String)
+
+  var isBase: Bool {
+    if case .Base = self {
+      return true
+    }
+
+    return false
+  }
+
+  var isNone: Bool {
+    if case .None = self {
+      return true
+    }
+
+    return false
+  }
+}
+
+extension Locale: CustomStringConvertible {
+  var description: String {
+    switch self {
+    case .None:
+      return ""
+
+    case .Base:
+      return "Base"
+
+    case .Language(let language):
+      return language
+    }
+  }
+}
+
+struct LocalizableStrings: WhiteListedExtensionsResourceType {
+  static let supportedExtensions: Set<String> = ["strings"]
+
+  let filename: String
+  let locale: Locale
+  let dictionary: [String : (value: String, params: [FormatSpecifier])]
+
+  init(url: NSURL) throws {
+    try LocalizableStrings.throwIfUnsupportedExtension(url.pathExtension)
+
+    guard let filename = url.filename else {
+      throw ResourceParsingError.ParsingFailed("Couldn't extract filename without extension from URL: \(url)")
+    }
+
+    // Get locale from url (second to last component)
+    var locale = Locale.None
```

Comment:
I feel a `let locale: Locale` that is set to `.None` in the `else` branch is nicer, since it's immutable and compiletime checks that we don't forget to set it to the correct value. Maybe even an custom initializer or the `rawValue` initializer could help us out?!